### ISSUE TITLE
Background image not functioning correctly

### ISSUE THREAD

**Issue by DNM1008** [microsoft/terminal#12128]
### Windows Terminal version

1.11.3471.0

### Windows build number

10.0.19044.0

### Steps to reproduce

Set the background in the setting section (Ctrl ,) by either choosing an image in the computer or the desktop background

### Expected Behavior

The background is changed to the image that I chose

### Actual Behavior

It returns a black background if I choose an image in the computer, and the wrong image when I choose desktop background (it was an image that I used a while back but changed recently.

My setting.json:

```
{
    "$schema": "https://aka.ms/terminal-profiles-schema",
    "actions": 
    [
        {
            "command": 
            {
                "action": "copy",
                "singleLine": false
            },
            "keys": "ctrl+c"
        },
        {
            "command": "paste",
            "keys": "ctrl+v"
        },
        {
            "command": "find",
            "keys": "ctrl+shift+f"
        },
        {
            "command": 
            {
                "action": "splitPane",
                "split": "auto",
                "splitMode": "duplicate"
            },
            "keys": "alt+shift+d"
        }
    ],
    "copyFormatting": "none",
    "copyOnSelect": false,
    "defaultProfile": "{574e775e-4f2a-5b96-ac1e-a2962a402336}",
    "profiles": 
    {
        "defaults": 
        {
            "backgroundImage": "D:\\Pictures\\Wallpapers\\908.918 Concept.jpg
![image](https://user-images.githubusercontent.com/54259825/148722817-ac31ae86-211f-4b7f-a627-a1e45ca4c229.png)
",
            "backgroundImageOpacity": 0.25,
            "padding": "0",
            "startingDirectory": "D:\\"
        },
        "list": 
        [
            {
                "commandline": "powershell.exe",
                "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
                "hidden": false,
                "name": "Windows PowerShell"
            },
            {
                "commandline": "cmd.exe",
                "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
                "hidden": false,
                "name": "Command Prompt"
            },
            {
                "guid": "{574e775e-4f2a-5b96-ac1e-a2962a402336}",
                "hidden": false,
                "name": "PowerShell",
                "source": "Windows.Terminal.PowershellCore"
            },
            {
                "guid": "{2c4de342-38b7-51cf-b940-2309a097f518}",
                "hidden": false,
                "name": "Ubuntu",
                "source": "Windows.Terminal.Wsl"
            },
            {
                "guid": "{b453ae62-4e3d-5e58-b989-0a998ec441b8}",
                "hidden": false,
                "name": "Azure Cloud Shell",
                "source": "Windows.Terminal.Azure"
            }
        ]
    },
    "schemes": 
    [
        {
            "background": "#0C0C0C",
            "black": "#0C0C0C",
            "blue": "#0037DA",
            "brightBlack": "#767676",
            "brightBlue": "#3B78FF",
            "brightCyan": "#61D6D6",
            "brightGreen": "#16C60C",
            "brightPurple": "#B4009E",
            "brightRed": "#E74856",
            "brightWhite": "#F2F2F2",
            "brightYellow": "#F9F1A5",
            "cursorColor": "#FFFFFF",
            "cyan": "#3A96DD",
            "foreground": "#CCCCCC",
            "green": "#13A10E",
            "name": "Campbell",
            "purple": "#881798",
            "red": "#C50F1F",
            "selectionBackground": "#FFFFFF",
            "white": "#CCCCCC",
            "yellow": "#C19C00"
        },
        {
            "background": "#012456",
            "black": "#0C0C0C",
            "blue": "#0037DA",
            "brightBlack": "#767676",
            "brightBlue": "#3B78FF",
            "brightCyan": "#61D6D6",
            "brightGreen": "#16C60C",
            "brightPurple": "#B4009E",
            "brightRed": "#E74856",
            "brightWhite": "#F2F2F2",
            "brightYellow": "#F9F1A5",
            "cursorColor": "#FFFFFF",
            "cyan": "#3A96DD",
            "foreground": "#CCCCCC",
            "green": "#13A10E",
            "name": "Campbell Powershell",
            "purple": "#881798",
            "red": "#C50F1F",
            "selectionBackground": "#FFFFFF",
            "white": "#CCCCCC",
            "yellow": "#C19C00"
        },
        {
            "background": "#282C34",
            "black": "#282C34",
            "blue": "#61AFEF",
            "brightBlack": "#5A6374",
            "brightBlue": "#61AFEF",
            "brightCyan": "#56B6C2",
            "brightGreen": "#98C379",
            "brightPurple": "#C678DD",
            "brightRed": "#E06C75",
            "brightWhite": "#DCDFE4",
            "brightYellow": "#E5C07B",
            "cursorColor": "#FFFFFF",
            "cyan": "#56B6C2",
            "foreground": "#DCDFE4",
            "green": "#98C379",
            "name": "One Half Dark",
            "purple": "#C678DD",
            "red": "#E06C75",
            "selectionBackground": "#FFFFFF",
            "white": "#DCDFE4",
            "yellow": "#E5C07B"
        },
        {
            "background": "#FAFAFA",
            "black": "#383A42",
            "blue": "#0184BC",
            "brightBlack": "#4F525D",
            "brightBlue": "#61AFEF",
            "brightCyan": "#56B5C1",
            "brightGreen": "#98C379",
            "brightPurple": "#C577DD",
            "brightRed": "#DF6C75",
            "brightWhite": "#FFFFFF",
            "brightYellow": "#E4C07A",
            "cursorColor": "#4F525D",
            "cyan": "#0997B3",
            "foreground": "#383A42",
            "green": "#50A14F",
            "name": "One Half Light",
            "purple": "#A626A4",
            "red": "#E45649",
            "selectionBackground": "#FFFFFF",
            "white": "#FAFAFA",
            "yellow": "#C18301"
        },
        {
            "background": "#002B36",
            "black": "#002B36",
            "blue": "#268BD2",
            "brightBlack": "#073642",
            "brightBlue": "#839496",
            "brightCyan": "#93A1A1",
            "brightGreen": "#586E75",
            "brightPurple": "#6C71C4",
            "brightRed": "#CB4B16",
            "brightWhite": "#FDF6E3",
            "brightYellow": "#657B83",
            "cursorColor": "#FFFFFF",
            "cyan": "#2AA198",
            "foreground": "#839496",
            "green": "#859900",
            "name": "Solarized Dark",
            "purple": "#D33682",
            "red": "#DC322F",
            "selectionBackground": "#FFFFFF",
            "white": "#EEE8D5",
            "yellow": "#B58900"
        },
        {
            "background": "#FDF6E3",
            "black": "#002B36",
            "blue": "#268BD2",
            "brightBlack": "#073642",
            "brightBlue": "#839496",
            "brightCyan": "#93A1A1",
            "brightGreen": "#586E75",
            "brightPurple": "#6C71C4",
            "brightRed": "#CB4B16",
            "brightWhite": "#FDF6E3",
            "brightYellow": "#657B83",
            "cursorColor": "#002B36",
            "cyan": "#2AA198",
            "foreground": "#657B83",
            "green": "#859900",
            "name": "Solarized Light",
            "purple": "#D33682",
            "red": "#DC322F",
            "selectionBackground": "#FFFFFF",
            "white": "#EEE8D5",
            "yellow": "#B58900"
        },
        {
            "background": "#000000",
            "black": "#000000",
            "blue": "#3465A4",
            "brightBlack": "#555753",
            "brightBlue": "#729FCF",
            "brightCyan": "#34E2E2",
            "brightGreen": "#8AE234",
            "brightPurple": "#AD7FA8",
            "brightRed": "#EF2929",
            "brightWhite": "#EEEEEC",
            "brightYellow": "#FCE94F",
            "cursorColor": "#FFFFFF",
            "cyan": "#06989A",
            "foreground": "#D3D7CF",
            "green": "#4E9A06",
            "name": "Tango Dark",
            "purple": "#75507B",
            "red": "#CC0000",
            "selectionBackground": "#FFFFFF",
            "white": "#D3D7CF",
            "yellow": "#C4A000"
        },
        {
            "background": "#FFFFFF",
            "black": "#000000",
            "blue": "#3465A4",
            "brightBlack": "#555753",
            "brightBlue": "#729FCF",
            "brightCyan": "#34E2E2",
            "brightGreen": "#8AE234",
            "brightPurple": "#AD7FA8",
            "brightRed": "#EF2929",
            "brightWhite": "#EEEEEC",
            "brightYellow": "#FCE94F",
            "cursorColor": "#000000",
            "cyan": "#06989A",
            "foreground": "#555753",
            "green": "#4E9A06",
            "name": "Tango Light",
            "purple": "#75507B",
            "red": "#CC0000",
            "selectionBackground": "#FFFFFF",
            "white": "#D3D7CF",
            "yellow": "#C4A000"
        },
        {
            "background": "#000000",
            "black": "#000000",
            "blue": "#000080",
            "brightBlack": "#808080",
            "brightBlue": "#0000FF",
            "brightCyan": "#00FFFF",
            "brightGreen": "#00FF00",
            "brightPurple": "#FF00FF",
            "brightRed": "#FF0000",
            "brightWhite": "#FFFFFF",
            "brightYellow": "#FFFF00",
            "cursorColor": "#FFFFFF",
            "cyan": "#008080",
            "foreground": "#C0C0C0",
            "green": "#008000",
            "name": "Vintage",
            "purple": "#800080",
            "red": "#800000",
            "selectionBackground": "#FFFFFF",
            "white": "#C0C0C0",
            "yellow": "#808000"
![908 918 Concept](https://user-images.githubusercontent.com/54259825/148722903-fddc6bff-015b-4215-a195-6154d4a50cd2.jpg)

        }
    ]
}
```
My terminal screenshot: 
![image](https://user-images.githubusercontent.com/54259825/148722855-d1f7cd6b-60b2-4d8a-9523-7de1068deecb.png)
The image that I wanted to set at background:
![908 918 Concept](https://user-images.githubusercontent.com/54259825/148722959-7e9ed0a7-9b42-452d-8d7c-2dbdf934a4d7.jpg)

**Comment by zadjii-msft** [comment]
I'm assuming that you don't actually have the following in your settings, and that was merely an artifact of pasting to GitHub, right?

```jsonc
            "backgroundImage": "D:\\Pictures\\Wallpapers\\908.918 Concept.jpg
![image](https://user-images.githubusercontent.com/54259825/148722817-ac31ae86-211f-4b7f-a627-a1e45ca4c229.png)
",
```

which should be 

```jsonc
            "backgroundImage": "D:\\Pictures\\Wallpapers\\908.918 Concept.jpg",
```
, right?


Is it only that file that doesn't work, or all image paths? Do you have a link to the actual image you're trying to use?

**Comment by DNM1008** [comment]
> I'm assuming that you don't actually have the following in your settings, and that was merely an artifact of pasting to GitHub, right?
> 
> ```js
>             "backgroundImage": "D:\\Pictures\\Wallpapers\\908.918 Concept.jpg
> ![image](https://user-images.githubusercontent.com/54259825/148722817-ac31ae86-211f-4b7f-a627-a1e45ca4c229.png)
> ",
> ```
> 
> which should be
> 
> ```js
>             "backgroundImage": "D:\\Pictures\\Wallpapers\\908.918 Concept.jpg",
> ```
> 
> , right?
> 
> Is it only that file that doesn't work, or all image paths? Do you have a link to the actual image you're trying to use?

Yes, my laptop lagged and I didn't realize.
Any other file is the same, I've tried different locations (in the C drive) and changed the name of the file so that there's no space or dot but still it's the same.

What's curious is the desktop background situation, it chose an image that I had as background a while ago

**Comment by ccfontes** [comment]
On my WT using WSL+Ubuntu 18.04, the background image also doesn't show because it's hidden by the "black" background. To prove this, when I increase window padding, the background image does show in the corners.

A way to mitigate this issue is to enable acrylic (with low opacity), but other windows in the background kind of also overlap.

PowerShell sessions doesn't have this issue.

**Comment by ToUNVRSe** [comment]
> 在我使用 WSL+Ubuntu 18.04 的 WT 上，背景图像也没有显示，因为它被“黑色”背景隐藏。为了证明这一点，当我增加窗口填充时，背景图像确实显示在角落里。
> 
> 缓解此问题的一种方法是启用亚克力（不透明度低），但背景中的其他窗口也会重叠。
> 
> PowerShell 会话没有这个问题。

It's strange, but disable ClearType is work

> On my WT using WSL+Ubuntu 18.04, the background image also doesn't show because it's hidden by the "black" background. To prove this, when I increase window padding, the background image does show in the corners.
> 
> A way to mitigate this issue is to enable acrylic (with low opacity), but other windows in the background kind of also overlap.
> 
> PowerShell sessions don't have this issue.

It's strange, but disable ClearType is work......

**Comment by DNM1008** [comment]
The solution I found was to upload the file that I wanted to set as background to Google Drive and then redownload it. If this was just 1 file, then perhaps there was a problem with metadata perhaps, but because it happened on a lot of files, I don't know

**Comment by thgross** [comment]
> It's strange, but disable ClearType is work......

Same problem here! I wondered why the background images don't show. Disabling
`"antialiasingMode": "cleartype"`
brings the background images back.

Windows Terminal version: 1.11.3471.0
Windows build number: 10.0.19044.0

**Comment by ccfontes** [comment]
Same here, disabling clearType antialiasing brings background image back, without resorting to acrylic.

Windows terminal: 1.12.10393.0
Ubuntu: 20.04.2 LTS
WSL: 2
Windows: 10 Enterprise build 19041.685

**Comment by ToUNVRSe** [comment]
> Same here, disabling clearType antialiasing brings background image back, without resorting to acrylic.
> 
> Windows terminal: 1.12.10393.0 Ubuntu: 20.04.2 LTS WSL: 2 Windows: 10 Enterprise build 19041.685

By the way, setting any value that less than 100 (such as 99) at acrylic can almost do the same on my WT.

**Comment by ccfontes** [comment]
> By the way, setting any value that less than 100 (such as 99) at acrylic can almost do the same on my WT.
True, I just tried 99 transparency acrylic with good results. Thanks.

Btw, I found Grayscale text antialiasing to be a good alternative to ClearType, when opting into not using acrylic / 99 transparency.

**Comment by zadjii-msft** [comment]
Oh yea I totally forgot about the cleartype thing. That was only fixed recently, in 1.13: #5703. I'm not sure that's actually the issue OP was hitting, but since they seemed to have sorted it out, I'm gonna close this for now. Every so often we get reports that a specific file on a user's computer doesn't work as a background image, but we've never been able to identify what's corrupted in the file to cause that. I think there's another issue floating around on the reop tracking that, so I'm not too worried we'll lose track of that.

**Comment by DNM1008** [comment]
It seems that the problem lies more in Windows not being able to read my
files, because changing my desktop background was an issue too. Programs
such as ImageGlass could read my images just fine, but somehow Windows
couldn’t.

Anyway uploading them to the internet and downloading them fixed the issue.

**Comment by ccfontes** [comment]
@DNM1008
I wonder if someone here could help you sort out the root problem related to the images, if you give more information about them, using some tool like ones mentioned here: https://superuser.com/questions/275502/how-to-get-information-about-an-image-picture-from-the-linux-command-line

Perhaps comparing image information before/after uploading images to the internet service could provide clues.

**Comment by DNM1008** [comment]
It was a while back so unfortunately I deleted the old photos :(

**Comment by zadjii-msft** [comment]
FWIW I had a crazy issue like this just the other night, trying to open an image in InDesign. InDesign absolutely refused to open any JPGs that I had downloaded from WhatsApp. However, opening them in another app and re-saving them fixed whatever encoding error there was in the file so it could be opened again. Wild.

I know that's seemingly totally unrelated, just thought it was weird that I noticed something similar in another app. Maybe there is something about certain JPG encodings that makes WinUI choke on the file and fail to load it? Hard to say for sure without a file that actively repros the issue, and obviously uploading the file here would likely re-encode the file without whatever artifact was causing the issue.
